app/scripts/utils/get-element-dim.js

```diff
@@ -1,5 +1,10 @@
 import { ElementResizeListener } from '../services';
 
+/**
+ * Get the dimensions of an element
+ * @param {HTMLElement} element - DOM element to get dimensions of.
+ * @return {[width: number, height: number]} The width and height of the element.
+ */
 const getElementDim = (element) => {
   ElementResizeListener.listen();
```

Comment:
I was really surprised to find this global side-effect in this "low-level" function.